gateway/1.4/modules/ROOT/pages/_partials/task-node-affinity.adoc

```diff
@@ -0,0 +1,223 @@
+//TEMPORARY TAG for test builds
+//tag::temp-tag1[]
+= Select a Node for Flex Gateway Deployments
+
+Flex Gateway is often critical to the security of a cluster, so it is important to host it on a specialized node that meets specific requirements, instead of using the nodes that host your other deployments.
+
+To select a suitable host node for a Pod that contains Flex Gateway, you can incorporate custom node affinity rules from your Pod specification (Pod spec) into an Ingress Controller Helm chart. Helm packages files for related Kubernetes resources into a Helm chart. For more information about node affinity, see the https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity[Kubernetes documentation^].
```

Comment:
regarding "Ingress Controller Helm chart":
Not sure what term we use for this. I see that term and "flex-gateway Helm chart". Do we have a preference? See examples here: https://docs.mulesoft.com/gateway/1.3/flex-conn-reg-run-up